core/src/edu/cornell/gdiac/main/GameController.java

```diff
@@ -104,33 +145,37 @@ public class GameController implements Screen {
    */
   private CollisionController collisionController;
   /**
-   * Whether or not this is an active controller
+   * Whether this is an active controller
    */
-  private boolean active;
+  private boolean screenActive;
   /**
-   * Whether we have completed this level
+   * Whether sound is on
    */
-  private boolean complete;
-  /**
-   * Whether we have failed at this world (and need a reset)
-   */
-  private boolean failed;
+  private boolean isSoundOn;
```

Comment:
Now that we're getting further in the game development process, I think we can completely replace the `IS_MUTED` variable with just the dynamic `isSoundOn` variable, so we just allow the user to change the sound by the menu.